packages/easy-email-core/src/utils/MjmlToJson.ts

```diff
@@ -136,7 +136,6 @@ export function MjmlToJson(data: MjmlBlockItem | string): IPage {
 
         // format padding
         formatPadding(blockData.attributes, 'padding');
-        formatPadding(blockData.attributes, 'inner-padding');
```

Comment:
I think the problem is more related to how I convert paddings internally.

A mistake had been made when handling padding types other than `padding`, in this case `inner-padding`

I think a quick solution is something like:
<img width="1064" alt="image" src="https://user-images.githubusercontent.com/20615214/160543566-b9dee778-5ac1-4e8b-bad2-98759d3a6be0.png">

Feel free to make changes and submit them.